evidence/characters/anemo/jean.md

```diff
@@ -1,91 +1,89 @@
 # Jean
 
-## Does Jean’s C4 Anemo Res Reduction still apply vertically if her burst misses mobs on Venti’s Q?
+## Jean's C4 verticality
 
-**By:** \(Special\#1775\)  
-**Added:** 11/24/2020
+**By:** Special\#1775
+<br/>**Added:** 11/24/2020
```

Comment:
Disagree with this, every other author line has a consistent style that is violated by this change